network/src/peers/peers.rs

```diff
@@ -113,17 +113,17 @@ impl Node {
             }
         }
 
-        // Attempt to connect to a few random bootnodes if the node has no active
-        // connections or if it's a bootnode itself.
-        if self.peer_book.get_active_peer_count() == 0 || self.config.is_bootnode() {
-            let random_bootnodes = self
+        // Attempt to connect to a few random peer provider nodes if the node has no active
```

Comment:
they were called "peer discovery nodes" earlier on, I think; it's best to either use a single name or even the `NodeType` in order to avoid confusion

Reply:
Alright, I'll use "beacon" (and clarify that refers to peer discovery nodes in the relevant docs/READMEs).